### PR TITLE
Bug Fix

### PR DESCRIPTION
**Pull request by g4ryy**
Changes made:
1. Fix bugs in user guide
2. Fix error messages
3. Remove traces of address book in the code and documentations
4. Update StudentUi to reflect changes in medical history label

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ title: User Guide
 ---
 ## Introduction
```
Comment (cliveong) [inline docs/UserGuide.md:5]: image

This shortcut does not seem to work

File: docs/UserGuide.md [hunk 2]
```diff
@@ -249,7 +259,15 @@ Check out how `involvement` is intended to be used in the glossary.
...
 
 <div markdown="span" class="alert alert-primary">:bulb: **Tip:**
-The app prevents the user from adding in duplicate students. Two students are duplicate if they have the same name and address.
+NewAddressBook prevents the user from adding in duplicate students. Two students are duplicate if they have the same name and address.
+However, this duplicate check is currently case-sensitive. For example, NewAddressBook will treat "John" and "john" as two
+different names as they have different casing. We will improve on this in future versions of NewAddressBook to allow this duplicate
+check to be case-insensitive.
+</div>
+
+<div markdown="span" class="alert alert-info">:information_source: **Notes about `FORM_CLASS`:**
```
Comment (cliveong) [inline docs/UserGuide.md:268]: image

The inline code not working, seems this is the case for all the other inline code inside tip, is this intentional?

File: src/main/java/seedu/address/logic/commands/ExitCommand.java [hunk 3]
```diff
@@ -9,7 +9,7 @@
 
     public static final String COMMAND_WORD = "exit";
 
-    public static final String MESSAGE_EXIT_ACKNOWLEDGEMENT = "Exiting Address Book as requested ...";
+    public static final String MESSAGE_EXIT_ACKNOWLEDGEMENT = "Exiting NewAdressBook as requested ...";
```
Comment (cliveong) [inline src/main/java/seedu/address/logic/commands/ExitCommand.java:12]: Typo on address

**Review by cliveong — COMMENTED** [review]
Minor issues, elses LGTM!

File: src/main/resources/view/StudentListCard.fxml [hunk 4]
```diff
@@ -31,7 +31,7 @@
                      <Insets left="4.0" />
                   </HBox.margin>
                </Label>
-            <Label fx:id="medicalHistory" ellipsisString="!!!" maxWidth="150.0" styleClass="cell_small_label" text="\$medicalHistory">
+            <Label fx:id="medicalHistory" ellipsisString="..." maxWidth="150.0" styleClass="cell_small_label" text="\$medicalHistory">
```
Comment (cliveong) [inline src/main/resources/view/StudentListCard.fxml:34]: Isnt the change to `!!!` intentional to indicate presence of medical issue?

File: docs/UserGuide.md (hunk 2 above)
Comment (g4ryy) [reply docs/UserGuide.md:268]: Yup it will only work in html

File: docs/UserGuide.md (hunk 1 above)
Comment (g4ryy) [reply docs/UserGuide.md:5]: Fixed!

File: src/main/java/seedu/address/logic/commands/ExitCommand.java (hunk 3 above)
Comment (g4ryy) [reply src/main/java/seedu/address/logic/commands/ExitCommand.java]: Fixed

File: docs/UserGuide.md (hunk 1 above)
Comment (g4ryy) [reply docs/UserGuide.md:5]: Fixed!

File: src/main/resources/view/StudentListCard.fxml (hunk 4 above)
Comment (g4ryy) [reply src/main/resources/view/StudentListCard.fxml:34]: This is considered as a feature change, so I changed it back

**Pull request merged**